torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16262,15 +16289,11 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
     ElementwiseUnaryPythonRefInfo(
         "_refs.trunc",
         torch_opinfo_name="trunc",
+        test_aliases=False,  # RuntimeError: no _refs support for torch.fix
     ),
     #
     # Elementwise Unary Special OpInfos
     #
-    ElementwiseUnaryPythonRefInfo(
-        "_refs.special.logit",
-        torch_opinfo_name="logit",
-        supports_nvfuser=False,
-    ),
```

Comment:
Shouldn't this one be the OG OpInfo, and then have `torch.logit` tested in terms of this?
This is a bit confusing, as it's not clear which one is the implementation and which one is the alias. At the moment, the implementation is in `refs.special.logit`, while the OpInfo is that of `torch.logit`.